src/app/static/src/app/components/downloadIndicator/DownloadIndicator.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-    <div>
+    <div id="indicator-download">
         <download-button
             :name="'downloadIndicator'"
             :disabled="downloadingIndicator"
-            @click="download"></download-button>
+            @trigger-download="download"></download-button>
```

Comment:
```suggestion
            @click.stop="download"></download-button>
```

Reply:
I think the issue here is that since Vue3 upgrade there is an new click event triggered from the `download-button` div. On current production (see click event)

![image](https://github.com/mrc-ide/hint/assets/39248272/a17c4232-2085-49b3-990b-2d67b818d8f3)

Whereas in current `hintpproofofconcept`

![Screenshot_20230621_125159](https://github.com/mrc-ide/hint/assets/39248272/c258a175-ee2f-473c-87f0-972300b70cd3)

In hintpproofofconcept there is a click event triggered from the `download-button` div which is new and not on production.

I think this is because of changes to fallthrough attributes. See [https://vuejs.org/guide/components/attrs.html#attribute-inheritance the click ](https://vuejs.org/guide/components/attrs.html#v-on-listener-inheritance) so I think what is happening here is when we click it is triggering an event from the button and from the div. You can see that as if we click in the div but not on the button we only get 1 download triggered. So I think what we want to do here is remove the click being triggered from the `download-button` div, and only keep the event trigger from the button - which we do want to propagate up to the `DownloadIndicator` component. 

It looks like we could do that by turning off fallthrough attributes by default and put them onto the button itself instead of the containing div, but that felt like quite a big hammer which could perhaps cause issues elsewhere. So opted for changing the name of the v-on so that click trigger is not created on the `download-button` div.